SECURITY.md

```diff
@@ -0,0 +1,18 @@
+# Security Policy
+
+## Supported Versions
+
+The following versions of ESMValTool are currently being supported with
+security updates.
+
+| Version  | Supported          |
+| -------- | ------------------ |
+| 2.10.0   | :white_check_mark: |
+| < 2.10.0 | :x:                |
+
+## Reporting a Vulnerability
+
+If you find a vulnerability, please contact the
+[ESMValTool Tech Lead Team][TLT].
+
+[TLT]: https://github.com/ESMValGroup/Community/tree/main/Tech%20Lead
```

Comment:
```suggestion
[TLT]: esmvaltool_tech_lead_team@listserv.dfn.de 
```
It is usual to write down an email address in these policies, that makes it a easy for people to contact us confidentially.

Reply:
Done in 06828e553 😊